src/cli.rs

```diff
@@ -527,6 +527,35 @@ pub fn build_cli() -> Command {
                         .help("Path to a Docker config.json-like path. Can be used to indicate registry authentication details")
                 )
         )
+        .subcommand(
+            Command::new("save")
+                .about("save policies in tar file")
+                .arg(
+                    Arg::new("policies")
+                        .num_args(1..)
+                        .required(true)
+                        .help("list of policies to save")
+                )
+                .arg(
+                    Arg::new("output")
+                    .long("output")
+                    .short('o')
+                    .required(true)
```

Comment:
Being this flag required, it means it's not possible to output the contents of the policy to the STDOUT, like `docker save` does

I'm fine not doing that

Reply:
We will just accept tar.gz files as agreed. Added to help message here https://github.com/kubewarden/kwctl/pull/348/commits/b1ca9ffbc94ee85ba0565cd4ca475ac4352cc69e